services/storage/src/simcore_service_storage/modules/celery/_task.py

```diff
@@ -1,57 +1,41 @@
 import asyncio
+import base64
 import inspect
 import logging
-import traceback
+import pickle
 from collections.abc import Callable, Coroutine
+from datetime import timedelta
 from functools import wraps
-from typing import Any, Concatenate, ParamSpec, TypeVar, overload
+from typing import Any, Concatenate, Final, ParamSpec, TypeVar, overload
 
-from celery import Celery  # type: ignore[import-untyped]
+from celery import Celery, Task  # type: ignore[import-untyped]
 from celery.contrib.abortable import AbortableTask  # type: ignore[import-untyped]
-from celery.exceptions import Ignore  # type: ignore[import-untyped]
+from pydantic import NonNegativeInt
 
 from . import get_event_loop
-from .models import TaskError, TaskId, TaskState
+from .models import TaskId
 from .utils import get_fastapi_app
 
 _logger = logging.getLogger(__name__)
 
-
-def error_handling(func: Callable[..., Any]) -> Callable[..., Any]:
-    @wraps(func)
-    def wrapper(task: AbortableTask, *args: Any, **kwargs: Any) -> Any:
-        try:
-            return func(task, *args, **kwargs)
-        except Exception as exc:
-            exc_type = type(exc).__name__
-            exc_message = f"{exc}"
-            exc_traceback = traceback.format_exc().split("\n")
-
-            _logger.exception(
-                "Task %s failed with exception: %s:%s",
-                task.request.id,
-                exc_type,
-                exc_message,
-            )
-
-            task.update_state(
-                state=TaskState.ERROR.upper(),
-                meta=TaskError(
-                    exc_type=exc_type,
-                    exc_msg=exc_message,
-                ).model_dump(mode="json"),
-                traceback=exc_traceback,
-            )
-            raise Ignore from exc  # ignore doing state updates
-
-    return wrapper
+_DEFAULT_TASK_TIMEOUT: Final[timedelta] = timedelta(minutes=1)
```

Comment:
why this value? what is the rationale?
a celery task is by default long running. also I saw you are overriding this everywhere. not sure it makes much sense.